### DISCUSSION TITLE
`cargo test` fails in `src-tauri`

### DISCUSSION THREAD

**Discussion by ruifengx** [tauri-apps/tauri#2511] [Q&A]
If I run `cargo test` in the `src-tauri` directory, cargo reports that procedural macro `tauri::generate_context!()` panicked with the following message:

    The `distDir` configuration is set to `"Set automatically by Vue CLI plugin"` but this path doesn't exist

How should I run tests for the Rust crate in `src-tauri`?

**Comment by nothingismagick** [comment]
It sounds like you are using the plugin. Please update your discussion with the required information about your installation using `tauri info`.

@nklayman we should dig into this.

**Reply by ruifengx** [reply]
Yes, I'm using the plugin for Vue CLI.

Below is what I get from `yarn tauri info`:

```
yarn run v1.22.11
$ path\to\project\node_modules\.bin\tauri info
 app:spawn Running "path\to\project\node_modules\@tauri-apps\cli\bin\tauri-cli.exe tauri info" +0ms


Operating System - Windows, version 10.0.19043 X64
Webview2 - 92.0.902.78

Node.js environment
  Node.js - 16.7.0
  @tauri-apps/cli - 1.0.0-beta.7
  @tauri-apps/api - Not installed

Global packages
  npm - 7.20.3
  yarn - 1.22.11

Rust environment
  rustc - 1.54.0
  cargo - 1.54.0

App directory structure
/.git
/.idea
/node_modules
/public
/src
/src-tauri

App
  tauri.rs - 1.0.0-beta.7
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  distDir - Set automatically by Vue CLI plugin
  devPath - Set automatically by Vue CLI plugin
  framework - Vue.js (Vue CLI)
  bundler - Webpack

Done in 8.77s.
```

I have no problem running `yarn tauri:serve` or `yarn tauri:build`, but I don't see a command for testing my Rust crate.
